src/apps/loan-list/materials/selectable-material/selectable-material.tsx

```diff
@@ -48,14 +51,24 @@ const SelectableMaterial: FC<SelectableMaterialProps & MaterialProps> = ({
   const t = useText();
 
   if (!item) return null;
-  const {
+  let {
     authorsShort = "",
     materialType,
     year = "",
     title = "",
     lang
   } = material || {};
 
+  // There is a chance material isn't provided, if corresponding reservation PID is
+  // incorrect (that's unfortunately possible). Then we use ilBibliographicRecord data.
+  if (!material) {
+    authorsShort = ilBibliographicRecord?.author || "";
+    materialType = "";
+    year = ilBibliographicRecord?.publicationDate || "";
+    title = ilBibliographicRecord?.title || "";
+    lang = ilBibliographicRecord?.language || "";
+  }
+
```

Comment:
Maybe use destructuring?

```suggestion
  if (!material) {
    const {
      author = "",
      publicationDate = "",
      title = "",
      language = ""
    } = ilBibliographicRecord || {};
  
    authorsShort = author;
    year = publicationDate;
    lang = language;
    materialType = "";
  }
```